### DISCUSSION TITLE
Freezes on file operation, restarting containers temporarily fixes

### DISCUSSION THREAD

**Discussion by kaibagley** [nextcloud/all-in-one#2918] [Questions] [answered]
Hey guys,

I've asked the question here with more details, but thought it'd be worth putting here for more exposure..
https://help.nextcloud.com/t/nextcloud-aio-freezes-have-to-manually-stop-all-containers-to-get-it-responsive/165826

~8 year old Dell XPS, running Ubuntu 22.04.
Docker compose, running calibre, calibre web, and AIO

I have 30GB of files i need to upload. I've spent 2 days uploading this and I'm currently 25/30GB done, uploading about 12 files each time. However it doesn't seem like the number of files i upload at a time effects it. All other operations on AIO are fine, going through settings, other apps, etc. but as soon as I look at my files, it feels like theres a 50% chance each second that my server just dies. RAM and CPU usage seem fine, looking at `docker stats`

This is super weird and annoying, I see people here using RPi's and such for this, but my laptop cant? I must be doing something wrong..

**Comment by szaimen** [comment, marked as answer]
Hi, can you for a test disable all optional addons and check if that makes it work?
